ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/css.txt

```diff
@@ -0,0 +1 @@
+less/header.less
```

Comment:
@Rajamouli-hcl : file name should be container.less

Reply:
updated to container.less